lib/measures/default_feature_reports/measure.rb

```diff
@@ -203,6 +242,682 @@ def run(runner, user_arguments)
     # DLM: measures like the GeoJSON to OSM measure can set this value
     feature_report.program.footprint_area = convert_units(floor_area, 'm^2', 'ft^2')
 
+    # maximum_number_of_stories
+    number_of_stories = building.standardsNumberOfStories.get if building.standardsNumberOfStories.is_initialized
+    number_of_stories ||= 1
+    puts "number of stories = #{number_of_stories}"
+    feature_report.program.maximum_number_of_stories = number_of_stories
+
+    # maximum_roof_height
+    floor_to_floor_height = building.nominalFloortoFloorHeight.to_f ## why floor_to_floor_height = 0 !?
+    puts "floor_to_floor_height = #{floor_to_floor_height}" 
+    maximum_roof_height = number_of_stories * floor_to_floor_height
+    feature_report.program.maximum_roof_height = maximum_roof_height
+
+    # maximum_number_of_stories_above_ground
+    number_of_stories_above_ground = building.standardsNumberOfAboveGroundStories.get if building.standardsNumberOfAboveGroundStories.is_initialized
+    number_of_stories_above_ground ||= 1
+    puts "number_of_stories_above_ground = #{number_of_stories_above_ground}"
+    feature_report.program.maximum_number_of_stories_above_ground = number_of_stories_above_ground
+
+    # maximum_number_of_parking_stories_above_ground
```

Comment:
number of parking spaces should be added as additional properties in this method, https://github.com/NREL/openstudio-standards/blob/053c02c6b91a84040ab61c42d51e03870eb09d33/lib/openstudio-standards/prototypes/common/objects/Prototype.Model.exterior_lights.rb#L7

work with @DavidGoldwasser to define the property name